rpcs3/Emu/SysCalls/Modules/cellGame.cpp

```diff
@@ -75,35 +76,16 @@ s32 cellHddGameCheck(PPUThread& ppu, u32 version, vm::cptr<char> dirName, u32 er
 	{
 		get->isNewData = CELL_HDDGAME_ISNEWDATA_NODIR;
 	}
-	else
-	{
-		// TODO: Is cellHddGameCheck really responsible for writing the information in get->getParam ? (If not, delete this else)
-		const auto& psf = psf::load(vfsFile("/dev_hdd0/game/" + dir + "/PARAM.SFO").VRead<char>());
-
-		get->getParam.parentalLevel = psf.at("PARENTAL_LEVEL").as_integer();
-		get->getParam.attribute = psf.at("ATTRIBUTE").as_integer();
-		get->getParam.resolution = psf.at("RESOLUTION").as_integer();
-		get->getParam.soundFormat = psf.at("SOUND_FORMAT").as_integer();
-		strcpy_trunc(get->getParam.title, psf.at("TITLE").as_string());
-		strcpy_trunc(get->getParam.dataVersion, psf.at("APP_VER").as_string());
-		strcpy_trunc(get->getParam.titleId, psf.at("TITLE_ID").as_string());
-
-		for (u32 i = 0; i < CELL_HDDGAME_SYSP_LANGUAGE_NUM; i++)
-		{
-			strcpy_trunc(get->getParam.titleLang[i], psf::get_string(psf, fmt::format("TITLE_%02d", i)));
-		}
-	}
-
-	// TODO ?
 
```

Comment:
I'm not sure. There really shouldn't be anything, but this function return results need to be confirmed on a PS3 probably. getParam is definitely not set by this function, though.